apps/web/src/pages/activities/ActivitiesPage.tsx

```diff
@@ -57,14 +78,34 @@ export function ActivitiesPage() {
     setToggleModal(false);
   }
 
+  const onBarClick = (item: IActivityGraphStats) => {
+    setValue('channels', item.channels, { shouldDirty: true });
+    setValue('templates', item.templates, { shouldDirty: true });
+    onFiltersChange({ channels: item.channels, templates: item.templates });
+  };
```

Comment:
when the chart bar is clicked set the `channels, templates` input values and update the filters